Description: build hardsid on kFreeBSD as well
 Make sure kFreeBSD is also detected as Linux.
 .
Author: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
Bug-Debian: http://bugs.debian.org/659276
---
Last-Update: <2012-02-17>

--- sidplay-libs-2.1.1.orig/configure.ac
+++ sidplay-libs-2.1.1/configure.ac
@@ -11,7 +11,7 @@ AC_CANONICAL_HOST
 
 hardsid=false
 case "$host" in
-    *linux*) hardsid=true
+    *linux*|*-k*bsd*-gnu) hardsid=true
         ;;
 esac
 AM_CONDITIONAL(HARDSID, test x$hardsid = xtrue)
